doc/lightning-parsefeerate.7.md

```diff
@@ -0,0 +1,45 @@
+lightning-parsefeerate -- Command for parsing a feerate string to a feerate
+===========================================================================
+
+SYNOPSIS
+--------
+
+**parsefeerate** *feerate_str*
+
+DESCRIPTION
+-----------
+
+The **parsefeerate** command returns the current feerate for any valid
+*feerate_str*. This is useful for finding the current feerate that a
+**fundpsbt** or **utxopsbt** command might use.
```

Comment:
This whole commit seems unnecessary?  `feerates` does this already?

Reply:
Not exactly, `feerates` returns the named current feerates; this takes any arbitrary feerate string and returns its value in perkw. I mean, technically, I could halfway parse the feerate string, figure out if it's a plain number or a named fee rate and if it's the latter match it up with the returned fields from `feerates`. Except that's a bit more work than just piping it through to lightningd to do it for me instead?